### DISCUSSION TITLE
blog/journey-of-computer-poisoning

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#38] [Announcements]
# blog/journey-of-computer-poisoning

哲学片段与学术之路

https://pathos.page/blog/journey-of-computer-poisoning

**Comment by ewader** [comment]
我也是好多年没有中过病毒了，还是要注意不在在不明网站下东西，微软自带的杀毒软件还是太感人了。话说，你用的评论系统也是giscus哈

**Comment by robustmaster** [comment]
教训中的第五点我试过，当时是用于存储高隐私度的文件，但太过于麻烦，没继续执行了。

**Comment by MarvinXu** [comment]
感谢分享，给我提了个醒！破解软件只敢在国内的论坛下载
